### PR TITLE
feat(80): created add helper function and tests for it

### PR DESCRIPTION
**Pull request by rhoofard**

### CODE REVIEW

File: pkg/receiver/gitproviderreceiver/internal/scraper/githubscraper/github_scraper.go [hunk 1]
```diff
@@ -265,7 +265,8 @@ func (ghs *githubScraper) scrape(ctx context.Context) (pmetric.Metrics, error) {
 				ghs.logger.Sugar().Errorf("error getting merged pull request count", zap.Error(err))
 			}
 
-			prPages := getNumPages(float64(100), float64(prOpenCount.Repository.PullRequests.TotalCount+prMergedCount.Repository.PullRequests.TotalCount))
+			totalPrCount := add(prOpenCount.Repository.PullRequests.TotalCount, prMergedCount.Repository.PullRequests.TotalCount)
```
Comment (adrielp) [inline pkg/receiver/gitproviderreceiver/internal/scraper/githubscraper/github_scraper.go:268]: TotalCount will always be of type `int` correct?

File: pkg/receiver/gitproviderreceiver/internal/scraper/githubscraper/graphql_helpers_test.go [hunk 2]
```diff
@@ -39,6 +39,38 @@ func TestGetNumPages1(t *testing.T) {
...
+	b := 10
+
+	expected := 20
+
+	num := add(a, b)
+
+	assert.Equal(t, expected, num)
+}
+
+func TestAdd1(t *testing.T) {
```
Comment (adrielp) [inline pkg/receiver/gitproviderreceiver/internal/scraper/githubscraper/graphql_helpers_test.go:64]: can we add a negative number with a postive number in the tests? I presume that github won't respond with a negative number

File: pkg/receiver/gitproviderreceiver/internal/scraper/githubscraper/github_scraper.go (hunk 1 above)
Comment (rhoofard) [reply pkg/receiver/gitproviderreceiver/internal/scraper/githubscraper/github_scraper.go:268]: yes since add is of type `int` but it should never be a partial value anyways.  I guess should add be changed to float so it can be used everywhere else then just be casted to `int` for totalPrCount?

File: pkg/receiver/gitproviderreceiver/internal/scraper/githubscraper/graphql_helpers_test.go (hunk 2 above)
Comment (rhoofard) [reply pkg/receiver/gitproviderreceiver/internal/scraper/githubscraper/graphql_helpers_test.go]: 👍

File: pkg/receiver/gitproviderreceiver/internal/scraper/githubscraper/github_scraper.go (hunk 1 above)
Comment (adrielp) [reply pkg/receiver/gitproviderreceiver/internal/scraper/githubscraper/github_scraper.go:268]: yea, that or a generic

**Review by adrielp — APPROVED** [review]
